### PR TITLE
Create opentelemetry-announced-support-for-profiling

### PR DESCRIPTION
**Pull request by cartermp**
staging until proper announcement time (tomorrow)

### CODE REVIEW

**Review by austinlparker — APPROVED** [review]

**Comment by cartermp** [comment]
@austinlparker I think technically if you merge it now it won't update until the 19th, but feel free to merge whenever you like (such as on stage maybe? your call)

**Pull request merged**
